package.json

```diff
@@ -64,13 +68,13 @@
 		"eslint-plugin-promise": "6.0.1",
 		"eslint-plugin-unicorn": "43.0.2",
 		"genversion": "3.1.1",
-		"jest": "29.0.3",
-		"jest-environment-node": "29.0.3",
+		"jest": "28.1.3",
+		"jest-environment-node": "28.1.3",
 		"jest-extended": "3.1.0",
 		"nodemon": "2.0.20",
 		"prettier": "2.7.1",
 		"prisma": "4.3.1",
-		"ts-jest": "29.0.2",
+		"ts-jest": "28.0.8",
```

Comment:
Just curious, why downgrade the versions of these packages? Do the newest versions have some kind of issue?

Reply:
When I made the last PR, I didn't check the versions I installed, since I thought 28 was still under active development. 29 was fine for the one empty file, but threw strange warnings in the console about `ts-jest`. As I wrote more tests, I found that 29 disagreed with TypeScript a ton, and wouldn't consume the config the way I'd laid it down. Seems there were some breaking changes from 28 to 29.

The downgrade fixed all of the issues.